public/js/util/validators.spec.js

```diff
@@ -1,19 +1,42 @@
-import {isHttpsUrl} from './validators';
+import { isHttpsUrl } from './validators';
 
-test('Should validate as HTTPS URL', () => {
-  let url = 'https://example.com';
 
-  return isHttpsUrl(url)
-    .then(res => {
-       expect(res).toBe(true);
-    });
-});
+describe("isHttpsUrl function", () => {
+
+  it('Should validate a simple URL as HTTPS URL', () => {
+    let url = 'https://example.com';
+
+    return isHttpsUrl(url)
```

Comment:
minor: `async`/`await` would make these tests slightly easier to read, but not sure what pattern is employed across the repository

Reply:
Agreed, although, it does look this is consistent with the rest of the project. I have removed the `return`s are they are unnecessary